spec/timezone.html

```diff
@@ -464,19 +470,34 @@ <h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-formattimezoneoffsetstring" aoid="FormatTimeZoneOffsetString">
-      <h1>FormatTimeZoneOffsetString ( _offsetNanoseconds_ )</h1>
+    <emu-clause id="sec-temporal-formatoffsettimezoneidentifier" type="abstract operation">
+      <h1>
+        FormatOffsetTimeZoneIdentifier (
+          _offsetNanoseconds_: an integer,
+          _style_: ~legacy~ or ~separated~
+        ): a String
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          It formats a time zone offset, in nanoseconds, into a UTC offset string.
+          If _style_ is ~legacy~, then the output will be in ±HHMM format.
+          If _style_ is ~separated~, then the output will be in ±HH:MM format if _offsetNanoseconds_ represents an integer number of minutes, and otherwise in ±HH:MM:SS[.S[S[S[S[S[S[S[S[S]]]]]]]]] format.
```

Comment:
Taking the relevant part from https://github.com/tc39/proposal-temporal/pull/2607#discussion_r1233185773 :
```suggestion
          If _style_ is ~legacy~, then the output will be formatted like ±HHMM.
          If _style_ is ~separated~, then the output will be formatted like ±HH:MM if _offsetNanoseconds_ represents an integer number of minutes, and otherwise like ±HH:MM:SS if _offsetNanoseconds_ represents an integer number of seconds and like ±HH:MM:SS.fff if it does not (where the "fff" part is a sequence of at least 1 and at most 9 fractional seconds digits with no trailing zeroes).
```

Reply:
Which I see from #2607 probably doesn't matter anyway.

Reply:
Yeah, given that it's being removed in #2607, I didn't bother replacing it here. But happy enough to put it back just in case #2607 isn't accepted for some reason.

Reply:
I made a slight tweak to the text for brevity:

> If _style_ is ~separated~, then the output will be formatted like ±HH:MM if _offsetNanoseconds_ represents an integer number of minutes, like ±HH:MM:SS if _offsetNanoseconds_ represents an integer number of seconds, and otherwise like ±HH:MM:SS.fff where "fff" is a sequence of at least 1 and at most 9 fractional seconds digits with no trailing zeroes.